webviz_subsurface/plugins/_inplace_volumes_onebyone.py

```diff
@@ -296,7 +296,11 @@ def plot_selector(self):
                     id=self.ids("plot-type"),
```

Comment:
Have you considered to switch to a` Dropdown` to make it more compact? An extra click to switch is of course a drawback, but it is already a lot of content in the view of this plugin, so think it would be nice to keep it as clean as possible.